db/seeds.rb

```diff
@@ -5,3 +5,12 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+# Clean out existing properties and user accounts. 
+User.destroy_all
```

Comment:
Is this going to work? are the dependencies all set up so it will destroy the associated files etc?

Reply:
I think it delete existing users when we run the server and then create new users as far as I understand.  I have checked the previous project and that's why I have written this line. if you think the line cause different error then we can delete it.